frontend/src/routes/Credentials/CredentialsForm.test.tsx

```diff
@@ -207,6 +207,8 @@ describe('add credentials page', () => {
                 cluster: 'cluster',
                 datacenter: 'datacenter',
                 defaultDatastore: 'defaultDatastore',
+                folder: 'folder',
+                resourcePool: 'resourcePool'
```

Comment:
Thanks for the contribution! Please let me know when a Bugzilla is opened and if you'd like to continue working on this PR or if you'd like us to put the finishing touches on it ourselves. Tests failed with:
```
[frontend] [prettier] [error] src/routes/Credentials/CredentialsForm.test.tsx: SyntaxError: ',' expected. (212:17)
[frontend] [prettier] [error]   210 |                 folder: 'folder',
[frontend] [prettier] [error]   211 |                 resourcePool: 'resourcePool'
[frontend] [prettier] [error] > 212 |                 imageContentSources: '',
[frontend] [prettier] [error]       |                 ^
[frontend] [prettier] [error]   213 |             },
[frontend] [prettier] [error]   214 |             true
[frontend] [prettier] [error]   215 |         )
```